stdlib/public/core/UnsafeRawPointer.swift.gyb

```diff
@@ -428,11 +428,11 @@ public struct Unsafe${Mutable}RawPointer : Strideable, Hashable, _Pointer {
   /// - Returns: A pointer to a newly allocated region of memory. The memory is
   ///   allocated, but not initialized.
   @_inlineable
-  public static func allocate(
-    bytes size: Int, alignedTo: Int
-  ) -> UnsafeMutableRawPointer {
+  public static func allocate(bytes: Int, alignedTo alignment: Int) 
```

Comment:
Nit: the original line breaking here was consistent with prevailing stdlib convention.